modules/ROOT/pages/inbound-message-flows.adoc

```diff
@@ -1,6 +1,10 @@
 = Inbound Message Flows
 
-An inbound message flow receives application messages from your partner, either directly or through a third-party connection; transforms the messages to the format used by your backend applications; and forwards the messages to those applications.
+An inbound message flow does the following:
+
+. Receives application messages from your partner, either directly or through a third-party connection.
```

Comment:
Should these have terminating punctuation?

Reply:
OK